checker/tests/value-index-interaction/MethodOverrides3.java

```diff
@@ -0,0 +1,17 @@
+// This class should not issues any errors, since these annotations are identical to the ones
+// on the class in java.io.PrintWriter.
```

Comment:
I found the 'on the class' confusing, as there are no annotations on the `PrintWriter` class.

Reply:
This comment is referring to the annotations in the Index JDK on `java.io.PrintWriter`. I'll clarify the comment.